### ISSUE TITLE
[Bug] Gallery slide width

### ISSUE THREAD

**Issue by dsedinkin** [VKCOM/VKUI#1337]
Откройте приложение на декстопе https://vk.com/app7695054

При открытии приложения, при листании слайдов они все равно не правильно выдают размер. После клика на одну из карточек, и возвращения обратно, все встает на свои места.
![image](https://user-images.githubusercontent.com/62930989/105996622-69d39180-60cc-11eb-928f-3af01f86a5dc.png)
![image](https://user-images.githubusercontent.com/62930989/105996638-70620900-60cc-11eb-88c4-aac4f0622f9e.png)
![image](https://user-images.githubusercontent.com/62930989/105996692-840d6f80-60cc-11eb-9638-2534d108ea7e.png)
![image](https://user-images.githubusercontent.com/62930989/105996783-9c7d8a00-60cc-11eb-9da3-78dfe2ae5350.png)
![image](https://user-images.githubusercontent.com/62930989/105996821-ab643c80-60cc-11eb-8620-cbce5af6391b.png)

**Comment by seadinkin2002** [comment]
v4.0.8 vkui

**Comment by gbowsky** [comment]
Пооткрывал в разных браузерах (Chrome и Safari последних версий), не смог воспроизвести проблему.

**Comment by seadinkin2002** [comment]
Воспроизвелось в Яндекс Браузере

**Comment by seadinkin2002** [comment]
Видео https://yadi.sk/i/NtCl0LgLoG8u7w

**Comment by ewgenius** [comment]
Проверил на большинстве браузеров, не воспроизвел ни на одном, в Яндекс браузере под виндой тоже

<img width="1293" alt="image" src="https://user-images.githubusercontent.com/827338/106106130-23c31000-6156-11eb-9dd5-d1c998cfef1b.png">

**Comment by ewgenius** [comment]
Была ли проблема на 4.0.7 и предыдущих версиях?

**Comment by seadinkin2002** [comment]
Была проблема в предыдущих версиях, думал что проблема решится в 4.0.8

**Comment by ewgenius** [comment]
В 4.0.8 исправили проблему когда размеры слайдов менялись при навигации
Тут возможно другая проблема при первой отрисовке приложения.

@seadinkin2002 есть возможность воспроизвести базовую структуру приложения с галереей на Codesandbox?

**Comment by seadinkin2002** [comment]
В таком случае есть предположение, раз у меня подгружаются слайды, а затем после обновления страницы встают на места, то проблема в этом. Щас постараюсь решить вопрос. В любом случае это тоже является багом)

**Comment by seadinkin2002** [comment]
Да, не обратил внимание, что по умолчанию слайд без установленной default размера. Но даже это не помогло 
![image](https://user-images.githubusercontent.com/62930989/106114908-f4ba9780-6171-11eb-9566-8e82e393211c.png)
  
![image](https://user-images.githubusercontent.com/62930989/106109821-c6d25480-616b-11eb-97d4-8071dc068187.png)

**Comment by seadinkin2002** [comment]
@ewgenius, смысл, вся работа демонстрируется в приложении

**Comment by ewgenius** [comment]
Воспроизвел - баг происходит из-за следующего:
- сначала в приложении не загружена лента, и нет скроллбара 
- после загрузки ленты, справа появляется скроллбар и сжимает контент на несколько пикселей, но это не воспринимается как resize event, поэтому галерея 

> нам не удавалось воспроизвести из-за плашки "включить уведомления" - из-за нее скролбар есть всегда, и ресайза не происходит


Посмотрим что можно сделать

**Comment by ewgenius** [comment]
Будет исправлено в #1367 

Если нужно временное решение - можно добавить `overflow: scroll` на body - скроллбар будет виден всегда, и ширина документа не будет скакать

**Comment by seadinkin2002** [comment]
На основании этого, изменил высоту приложения и баг исправился. Спасибо

**Comment by seadinkin2002** [comment]
Сейчас снизу контент сам по себе есть, и scrollbar при запуске приложения присутствует

**Comment by EldarMuhamethanov** [comment]
Кажется баг сильно устарел. У меня не получилось воспроизвести. Как я понял проблема была в расайзе контента и неправильном пересчете размера слайдов. Сейчас, если такая проблема воспроизводится, то можно воспользоваться свойство `resizeSource="element"`(добавлен в #8055), в таком случае пересчет размеров слайдов будет происходить при изменении размера самой галереи
